src/k-provider/ovp/provider.js

```diff
@@ -70,6 +70,27 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
     });
   }
 
+  doRequest(loaders: Array<ExternalRequestLoader>, params: any = {}): Promise<any> {
+    const dataLoader = new OVPDataLoaderManager(this.playerVersion, this.partnerId, params.ks || this.ks, this._networkRetryConfig);
```

Comment:
Do we need to pass the KS? Can't we rely on the ks in the provider?

Reply:
As agreed we will use the ks from the provider and if needed the external call will add the ks to the specific service request